magpie-check-inputs

```diff
@@ -1502,6 +1519,62 @@ then
 fi
 
 #
+# Check Zeppelin Inputs
+#
+
+if [ "${ZEPPELIN_SETUP}" == "yes" ]
+then
+    if [ "${HADOOP_SETUP}" != "yes" ]
+    then
+	echo "Zeppelin requires Hadoop to be setup, set HADOOP_SETUP to yes"
+	exit 1
+    fi
+
```

Comment:
does zeppelin require hadoop?  It's not listed in the README instructions above.